### DISCUSSION TITLE
Finde den SAE im SHM2.0 einfach nicht

### DISCUSSION THREAD

**Discussion by Kofferschluessel** [camueller/SmartApplianceEnabler#535] [Installation und Betrieb]
Also es tut mir leid das ich jetzt hier anfange zu schreiben aber ich weiß nicht mehr weiter.
Hänge den ganzen Tag schon hier und versuche den SAE im HM zu finden.
Installation automatisch auf dem Raspi, SAE läuft.
Aber er taucht im SHM2.0 einfach nicht auf.
Bitte gebt mir Tipps was ich noch tun kann. Bzw. Befehle oder Fragen.
Die Anleitungen usw. habe ich alle schon überflogen. Tagsüber bin ich wegen einem Burnout in der Klinik und abends versuche ich jetzt mein System hier hinzukriegen.

Eine SEMP-Abfrage zu meinem Gerät bringt
F-00000001-000000000001-00 SMI53M75EU/61 DishWasher 111 Bosch 2400 Measurement false false true F-00000001-000000000001-00 true Off 0 0 60 F-00000001-000000000001-00 0 1635 1200 1635 F-00000001-000000000001-00 30435 88035 1200 12600 F-00000001-000000000001-00 116835 174435 1200 12600

Im Anlagenlogbuch habe ich noch gefunden:

  | 14.06.2023 21:23:21 | SHM 2 230 | EM-Gateway gefunden: SEMP, Name: SmartApplian

Aber bei der Neugerätesuche zeigt es null an.

Was mir der SHM auch immer anzeigt ist:
Das von Ihrem Sunny Home Manager gefundene Gerät mit der Seriennummer xxxxxx817 ist bereits in einer anderen Anlage registriert und kann deshalb Ihrer Anlage nicht hinzugefügt werden.

Das kommt schon den ganzen Abend. Kann darin ein Problem liegen? In der alten Anlage ist aber nichts registriert, denn der alte HM war schon lange weg.

**Comment by camueller** [comment]
Die von Dir gewählte ID für das Gerät ist bereits vergeben. Siehe https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Appliance_DE.md#id

**Reply by Kofferschluessel** [reply]
Super, vielen Dank. Genau das war die Lösung. Hab dann den Text irgendwie falsch verstanden gehabt. Eine wirre Reihenfolge an Zahlen vergeben und 60s später war der Geschirrspüler da.

**Comment by Kofferschluessel** [comment]
Nochmal blöde Frage, mir zeigt es den Verbrauch vom Geschirrspüler im Sunny Portal nicht an. Wenn er läuft, seh ich die verbrauchte Leistung. Aber im Portal ist der Gesamtverbrauch oder selbst der Nutzungsverlauf nicht ersichtlich.

Hab ich wieder was überlesen? Möchte nämlich noch Waschmaschine, Trockner mit Steuerung einbinden und die Wärmepumpe und Wallbox über 3EM als Zähler.

![Portal](https://github.com/camueller/SmartApplianceEnabler/assets/136642763/11867e8a-5692-401c-89ba-45764690aff2)
![Schalter Einstellung](https://github.com/camueller/SmartApplianceEnabler/assets/136642763/61b7a0f2-13ec-442e-beec-64fa4ad1effd)
![Schaltzeiten](https://github.com/camueller/SmartApplianceEnabler/assets/136642763/0d2a249a-6866-4c82-85bc-d3ff57dbcf00)
![Spüler Einstellung](https://github.com/camueller/SmartApplianceEnabler/assets/136642763/4592ef66-6ea9-4b1c-90e2-a62386a998d6)
![Zähler Einstellung](https://github.com/camueller/SmartApplianceEnabler/assets/136642763/0cfbe5fa-e333-4ebe-9976-41ce065814ea)

**Comment by camueller** [comment]
Zweiter Punkt: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/QA_DE.md#semp2
